packages/kbn-lens-embeddable-utils/README.md

```diff
@@ -1,241 +1,10 @@
-        label: 'Disk Read Throughput',
-        value: "average(system.load.1) / max(system.load.cores)",
-        format: {
-          id: 'percent',
-          params: {
-            decimals: 1,
-          },
-        },
-      }],
-      options: {
-      buckets: {type: 'date_histogram'},
-      },
-    }),
-    new XYReferenceLineLayer({
-      data: [{
-
-        value: "1",
-        format: {
-          id: 'percent',
-          params: {
-            decimals: 1,
-          },
-        },
-      }],
-    }),
-  ],
-  dataView,
-  formulaAPI
-});
-```
-
-#### Adding Multiple Data Sources in the Same Layer
-
-In an XY chart, it's possible to define multiple data sources within the same layer.
-
-To configure multiple data sources in an XY data layer, simply provide an array of data to the same YXDataLayer class:
-
-```ts
-const xyChart = new XYChart({
-  layers: new YXDataLayer({
-    data: [{
-      label: 'RX',
-      value: "average(host.network.ingress.bytes) * 8 / (max(metricset.period, kql='host.network.ingress.bytes: *') / 1000)",
-      format: {
-        id: 'bits',
-        params: {
-          decimals: 1,
-        },
-      },
-    },{
-      label: 'TX',
-      value: "(average(host.network.egresss.bytes) * 8 / (max(metricset.period, kql='host.network.egresss.bytes: *') / 1000)",
-      format: {
-        id: 'bits',
-        params: {
-          decimals: 1,
-        },
-      },
-    }],
-    options: {
-      buckets: {type: 'date_histogram'},
-    },
-  }),
-  dataView,
-  formulaAPI
-});
-```
-
-#### Building Lens Chart Attributes
-
-The `LensAttributesBuilder` is responsible for creating the full JSON object that combines the attributes returned by the chart classes. Here's an example:
-
-```ts
-const builder = new LensAttributesBuilder({ visualization: xyChart });
-const attributes = builder.build();
-```
-
-The `attributes` object contains the final JSON representation of the chart configuration and can be used to render the chart with Lens.
-
-### Usage with Lens EmbeddableComponent
-
-To display the charts rendered with the Lens Attributes Builder, it's recommended to use the Lens `EmbeddableComponent`. The `EmbeddableComponent` abstracts some of the chart styling and other details that would be challenging to handle directly with the Lens Attributes Builder.
-
-```tsx
-const builder = new LensAttributesBuilder({
-  visualization: new MetricChart({
-    layers: new MetricLayer({
-      data: {
-        label: 'Disk Read Throughput',
-        value: "counter_rate(max(system.diskio.read.count), kql='system.diskio.read.count: *')",
-        format: {
-          id: 'bytes',
-          params: {
-            decimals: 1,
-          },
-        },
-      },
-    }),
-    dataView,
-    formulaAPI
-  }),
-});
-
-const lensAttributes = builder.build();
-
-<EmbeddableComponent
-  attributes={lensAttributes}
-  viewMode={ViewMode.VIEW}
-  ...
-/>
-```
\ No newline at end of file
+Check kibana developer docs ...
```

Comment:
Can you put a link to it here?

Reply:
can we put internal links here ? for external ppl they would need to build it themselfs

Reply:
https://docs.elastic.dev/kibana-dev-docs/lens/config-builder

Reply:
Yes, I see the ML packages have it: `x-pack/packages/ml/agg_utils/README.md`.

I've also noticed how the ML package have the `homepage` entry in `the package.json`:

```
...
  "homepage": "https://docs.elastic.dev/kibana-dev-docs/lens/config-builder",
```